scanhub-ui/src/components/ExamModifyModal.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (C) 2024, BRAIN-LINK UG (haftungsbeschränkt). All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-ScanHub-Commercial
+ *
+ * ExamModifyModal.tsx is responsible for rendering a modal with an interface
+ * to modify an existing exam.
+ */
+import Button from '@mui/joy/Button'
+import FormLabel from '@mui/joy/FormLabel'
+import Grid from '@mui/joy/Grid'
+import Input from '@mui/joy/Input'
+import Modal from '@mui/joy/Modal'
+import ModalClose from '@mui/joy/ModalClose'
+import ModalDialog from '@mui/joy/ModalDialog'
+import Stack from '@mui/joy/Stack'
+import Typography from '@mui/joy/Typography'
+import * as React from 'react'
+import { useMutation } from 'react-query'
+
+import LoginContext from '../LoginContext'
+import { examApi } from '../api'
+import { BaseExam, ExamOut } from '../generated-client/exam'
+import { ModalPropsModify } from '../interfaces/components.interface'
+import NotificationContext from '../NotificationContext'
+
+
+const formContent: {key: keyof BaseExam, label: string}[] = [
```

Comment:
We should allow to update the status

Reply:
At the moment I think the status will probably be updated automatically when executing the exam and I would not want the user to interfere with such a mechanism. However I also have the feeling that at some point we might want to add additional options for an administrator to fix problematic situations like "examined patient under wrong name" or similar.